libobs/media-io/video-io.c

```diff
@@ -310,6 +312,7 @@ static enum video_colorspace collapse_space(enum video_colorspace cs)
 	case VIDEO_CS_2100_HLG:
 		cs = VIDEO_CS_2100_PQ;
 	}
+	PRAGMA_WARN_POP
```

Comment:
Instead of assigning to `cs`, I'd like to suggest just returning from each case condition.
```c
static enum video_colorspace collapse_space(enum video_colorspace cs)
{
	switch (cs) {
	case VIDEO_CS_DEFAULT:
	case VIDEO_CS_SRGB:
		return VIDEO_CS_709;
	case VIDEO_CS_2100_HLG:
		return VIDEO_CS_2100_PQ;
	default:
		return cs;
	}
}
```

Reply:
`default` is a no go there, it seems to need to be all cases in the future. But for now, in this PR it's pragma.